AppCenterAnalytics/AppCenterAnalytics/TransmissionTarget/MSPropertyConfigurator.m

```diff
@@ -60,7 +63,7 @@ - (void)removeEventPropertyForKey:(NSString *)propertyKey {
       MSLogError([MSAnalytics logTag], @"Event property key to remove cannot be nil.");
       return;
     }
-    [self.eventProperties removeObjectForKey:propertyKey];
+    [self.eventProperties.properties removeObjectForKey:propertyKey];
```

Comment:
You can use the remove method form the MSEventProperties directly.

Reply:
There is no remove method on MSEventProperties unless I am missing something @clpolet

Reply:
Right I got confused.